src/@context/Profile.tsx

```diff
@@ -224,12 +224,8 @@ function ProfileProvider({
       const tokenOrders = await getUserTokenOrders(accountId, chainIds)
 
       for (let i = 0; i < tokenOrders?.length; i++) {
-        const did = web3.utils
-          .toChecksumAddress(tokenOrders[i].datatoken.address)
-          .replace('0x', 'did:op:')
-        didList.push(did)
+        didList.push(tokenOrders[i].datatoken.address)
```

Comment:
as far as I see, this is not a DID list anymore but a list of Datatoken addresses, so the variable should be named differently